tests/functional/api/test_profile.py

```diff
@@ -20,7 +20,7 @@ def test_it_returns_profile_for_authenticated_user(self, app, user_with_token):
 
         user, token = user_with_token
 
-        headers = {"Authorization": str("Bearer {}".format(token.value))}
+        headers = {"Authorization": str(f"Bearer {token.value}")}
```

Comment:
What is this `str` about? Isn't an f-string a str?

This looks like it might be a hang over from Python 2

Reply:
Remove most of those now I think.